### PR TITLE
Untangle new header

### PR DESCRIPTION
**Pull request by i-like-robots**
This pulls the nested dependencies of the new header and navigation components up to this level to make their eventual abstraction them out of n-header-footer easier.

https://github.com/Financial-Times/n-header-footer/pull/241

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -53,6 +46,7 @@
     "superstore-sync": "^2.1.0"
   },
   "resolutions": {
-    "o-viewport": "^2.0.0"
+    "o-viewport": "^2.0.0",
+    "n-header-footer": "^3.0.0"
```
Comment (i-like-robots) [inline bower.json]: chicken and egg

File: bower.json [hunk 2]
```diff
@@ -32,13 +31,7 @@
     "next-myft-client": "^6.4.2",
     "o-ads": "^4.0.0",
     "o-buttons": "^4.0.1",
-    "n-message-prompts": "^2.1.8",
```
Comment (wheresrhys) [inline bower.json]: I don't get it - why you removing these?

Comment (i-like-robots) [inline bower.json]: I didn't! I can only assume this happened when I asked Bower to persist my resolution decision.

Comment (i-like-robots) [inline bower.json]: Bash history:

```
gco matth/untangle-new-header
bower i
git add bower.json
git commit --amend
git push origin matth/untangle-new-header -f
```

Comment (i-like-robots) [inline bower.json]: https://github.com/Financial-Times/n-ui/pull/82

**Pull request merged**